apps/site/pages/docs/getting-started/3-project-structure.mdx

```diff
@@ -52,24 +54,26 @@ Let's start taking a look at the structure that represents the source code of a
 
 - `cypress.config.ts` - Configures the options and preferences required for running tests with Cypress.
 
-- [`faststore.config.js`](/docs/faststore-config-options) - Configures aspects of the project such as SEO settings, store ID, and theme using this file. Keep in mind that some of these configurations are set during the [FastStore Onboarding](/docs/getting-started/1-faststore-onboarding/overview) process. For more information on the configuration options available in this file, refer to [Configuration options for `faststore.config.js`](/docs/faststore-config-options)
+- [`faststore.config.js`](/docs/faststore-config-options) - Configures aspects of the project such as SEO preferences, VTEX accout options, and theme. Note that these configurations are set during the [onboarding process](/docs/getting-started/1-faststore-onboarding/overview), reducing the need for changes to this file.
+For more information on configuratioon options for this file, refer to [Configuration options for faststore.config.js](/docs/faststore-config-options).
 
 - `next-env.d.ts` - Generates a TypeScript definition file for Next.js.
-
+  
 - `tsconfig.json` - Configures the compiler options and project settings for TypeScript, including target version, module resolution, and source file locations.
 
-- `vtex.env` - Stores environment variables that your application can access during runtime, such as API keys and database credentials.
+- `vtex.env` - Holds environment variables that your application can access during runtime, such as API keys and database credentials. 
+
+- `yarn.lock` - Locks down the exact versions of your project's dependencies, including transitive dependencies, to ensure consistent builds across different machines and environments. This file is automatically generated based on the dependencies in your `package.json` file.
 
-- `yarn.lock` - Locks down the exact versions of your project's dependencies, including transitive dependencies, to ensure consistent builds across different machines and environments. This file is automatically generated by Yarn based on the dependencies in your package.json file.
+###  `package.json` 
+Defines the project's metadata and handles dependencies and scripts using [Yarn](https://yarnpkg.com/migration/overview). Let's take a closer look at three important packages declared in this file:
 
-###  `package.json`
-Defines the project's metadata and handles dependencies and scripts using yarn. Let's take a closer look at three important packages inside this file:
+#### `@faststore/core`
+Bundles FastStore source code, including the FatsStore starter: `starter store`. It contains three sub-packages, `Components`, `SDK`, and `API`, which handle the starter.
 
-- #### `@faststore/core`
-Bundles FastStore source code, including the starter for FatsStore projects: `starter store`. It contains three sub-packages, `Components`, `SDK`, and `API`, which handle the starter.
+#### `@faststore/ui` 
```

Comment:
I think we can remove this, nowadays, we're not declaring this package in the package.json. It's being updated through `@faststore/core`